molo/profiles/templates/profiles/change_password.html

```diff
@@ -2,41 +2,43 @@
 {% load i18n %}
 
 {% block content %}
-<div class="form-wrap change-password">
-<form method="post" id="frmProfilePasswordChange" action="" class="profile">
+<div class="profiles">
+<form method="post" action="" class="profiles__form">
 {% csrf_token %}
    {% if form.errors or form.non_field_errors %}
       {% if form.non_field_errors %}
           {% for error in form.non_field_errors %}
-              <p class="errorlist">{{ error }}</p>
+              <span class="error">{{ error }}</span>
           {% endfor %}
       {% endif %}
   {% endif %}
 
   {% if messages %}
-    <ul class="errorlist">
+    <ul class="error-list">
         {% for message in messages %}
-        <li>{{ message }}</li>
+        <li class="error-list__item error">{{ message }}</li>
         {% endfor %}
     </ul>
   {% endif %}
 
-   <div>
-      <label class="label-wide">{% trans "Old Password:" %}</label>
-      <p>{{form.old_password}}</p>
-      <span>{{form.old_password.errors}}</span>
-   </div>
-   <div>
-      <label class="label-wide">{% trans "New Password:" %}</label>
-      <p>{{form.new_password}}</p>
-      <span>{{form.new_password.errors}}</span>
-   </div>
-   <div>
-      <label class="label-wide">{% trans "Confirm New Password:" %}</label>
-      <p>{{form.confirm_password}}</p>
-      <span>{{form.confirm_password.errors}}</span>
-   </div>
-   <input type="submit" value='{% trans "Change Password" %}' class="btn">
+  <fieldset>
+      <label>{% trans "Old Password" %}</label>
+      {{form.old_password}}
+      {{form.old_password.errors}}
```

Comment:
This is already has a ul list but I don't know how the error class is applied.

With the span 
<img width="615" alt="screen shot 2016-06-29 at 4 39 25 pm" src="https://cloud.githubusercontent.com/assets/1453355/16456492/4c9af3be-3e18-11e6-89b3-7462f841eb0a.png">

Without the span 
<img width="625" alt="screen shot 2016-06-29 at 4 40 13 pm" src="https://cloud.githubusercontent.com/assets/1453355/16456514/6035d5e2-3e18-11e6-94dc-0c0075153ca8.png">